vignettes/function-flow.Rmd

```diff
@@ -92,12 +92,26 @@ outputs.](images/function-flow-population.png)
 
 ### Inclusion events
 
-#### HbA1c tests above 48 mmol/mol
+```{r Add algorithm data, include=FALSE}
+library(dplyr); library(knitr)
+load("../data/algorithm.Rda")
```

Comment:
````suggestion
```{r, include=FALSE}
library(dplyr)
library(osdc)
````

Since the `algorithm` dataset is stored in `data/`, whenever you load `osdc` you also gain access to those datasets so you don't need to load them :)